sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2215,7 +2219,11 @@ case class DatePart(field: Expression, source: Expression, child: Expression)
       > SELECT _FUNC_(seconds FROM interval 5 hours 30 seconds 1 milliseconds 1 microseconds);
        30.001001
   """,
+  note = """
+    The _FUNC_ function is equivalent to `date_part`.
```

Comment:
`date_part(field, source)`

Reply:
BTW is `EXTRACT` more widely used? If yes then we should put the document in `Extract`.

Reply:
sgtm